driver-core/src/main/com/mongodb/internal/bulk/IndexRequest.java

```diff
@@ -536,4 +537,30 @@ public IndexRequest wildcardProjection(final BsonDocument wildcardProjection) {
         this.wildcardProjection = wildcardProjection;
         return this;
     }
+
+    /**
+     * Gets if the index should exist on the target collection but should not be used by the query
+     * planner when executing operations.
+     *
+     * @return true if the index should not be used by the query planner when executing operations.
+     * @mongodb.server.release 4.4
```

Comment:
Methods in internal comments don't need these last two annotations, and definitely shouldn't have \@since, since it implies some sort of API guaranteed

Reply:
I have removed the annotations from these new methods. At some point, the annotations should be removed from the other methods in this class.

Reply:
Agreed.  These classes used to be part of the API but as of 4.0 no longer are